src/dpl/src/Opendp.cpp

```diff
@@ -389,6 +385,10 @@ bool Opendp::isStdCell(const Cell* cell) const
     case dbMasterType::ENDCAP_LEF58_LEFTBOTTOMCORNER:
     case dbMasterType::ENDCAP_LEF58_RIGHTTOPCORNER:
     case dbMasterType::ENDCAP_LEF58_LEFTTOPCORNER:
+      return true;
+    case dbMasterType::BLOCK:
+    case dbMasterType::BLOCK_BLACKBOX:
+    case dbMasterType::BLOCK_SOFT:
       // These classes are completely ignored by the placer.
```

Comment:
You should move this comment above your changes.

Reply:
Why?

Reply:
Well they are now grouped with the COVER, RING and PAD classes. If we're doing that it seems like that comment would apply to them to. If it doesn't it almost feels like we should give them their own return value to differentiate it from the classes below.

Reply:
Endcaps were always above that comment so nothing has changed in regards to it.  I just moved them to the 'return true' section.  They are not "grouped with the COVER, RING and PAD classes"

Reply:
Seems fine then.